src/ControlSystem/Protocols/Submeasurement.hpp

```diff
@@ -0,0 +1,65 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <type_traits>
+
+#include "Utilities/TMPL.hpp"
+
+namespace control_system::protocols {
+/// \brief Definition of a portion of a measurement for the control
+/// systems
+///
+/// These structs are referenced from structs conforming to the
+/// Measurement protocol.  They define independent parts of a control
+/// system measurement, such as individual horizon-finds in a
+/// two-horizon measurement.
+///
+/// A conforming struct must provide an `interpolation_target_tag`
+/// type alias templated on the \ref ControlSystem "control systems"
+/// using this submeasurement.  (This template parameter must be used
+/// in the call to `RunCallbacks` discussed below.)  This alias may be
+/// `void` if the submeasurement does not use an interpolation target
+/// tag.  This is only used to collect the tags that must be
+/// registered in the metavariables.
+///
+/// A struct must also define an `argument_tags` type list and an
+/// `apply` function taking the corresponding arguments followed by
+///
+/// - `const LinkedMessageId<double>& measurement_id`
+/// - `Parallel::GlobalCache<Metavariables>& cache`
+/// - `const /*component-defined*/& array_index`
+/// - `const ParallelComponent* /*meta*/`
+/// - `ControlSystems /*meta*/`
+///
+/// where the last two are generally template parameters to the
+/// function.  This function will be called on every element, and
+/// these calls must collectively result in a single call on one chare
+/// (which need not be one of the element chares) to
+/// `control_system::RunCallbacks<ConformingStructBeingDefinedHere,
+/// ControlSystems>::apply`.  This will almost always require
```

Comment:
Docs complain about this link to `control_system::RunCallbacks<ConformingStructBeingDefinedHere, ControlSystems>::apply`. I'm not sure how to fix this though

Reply:
I'm hoping someone else will have an idea.  It renders fine.

Reply:
And this time it passed?  Maybe something from rebasing on some of the recent doxygen changes.